### PR TITLE
Add nilknarfgnow file

### PR DESCRIPTION
**Pull request by nilknarfgnow**
Added stuff, and delete @tbujas 

When is lunch, @ningguodfs ?

### CODE REVIEW

File: nilknarfgnow-arkansas.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Searcy, AR
```
Comment (tbujas) [inline nilknarfgnow-arkansas.md:1]: @nilknarfgnow leaving an unimaginative comment.

File: nilknarfgnow-arkansas.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+### Restaurants
```
Comment (tbujas) [inline nilknarfgnow-arkansas.md:3]: @nilknarfgnow Know any good restaurants there?

Comment (nilknarfgnow) [reply nilknarfgnow-arkansas.md:3]: Absolutely none :sob:

**Review by nilknarfgnow — COMMENTED** [review]
Good

**Review by pjbrown9 — COMMENTED** [review]
Looks Great!

**Review by ningguodfs — APPROVED** [review]
review done

**Review by pjbrown9 — APPROVED** [review]

**Pull request merged**